src/appkit.cs

```diff
@@ -18725,7 +18729,9 @@ interface NSToolbarItemGroup
 		NSToolbarItem[] Subitems { get; set; }
 	}
 
+#if XAMCORE_4_0
```

Comment:
It's better to leave this as `[DisableDefaultCtor]` in the bindings.
Next add a `NSTouch.cs` file and add a `[Obsolete ("This constructor does not create a valid native instance, please use ...")]` on an default .ctor that is under `#if !XAMCORE_4_0`
That will give enough clues for developers to avoid the API (and the weird issues it could cause if used)